src/components/ContentWrap/ContentWrap.js

```diff
@@ -2,14 +2,83 @@
  * This function returns a div that contains the left and right columns for issuer
  * @returns An empty element with the left and right column inside.
  */
+import { useEffect, useState } from 'react'
 import ColumnLeftWrap from './../ColumnLeft/ColumnLeftWrap'
 import ColumnRightWrap from './../ColumnRight/ColumnRightWrap'
+import useGetLoopedPresentProofRecords from '../../interface/hooks/use-get-looped-present-proof-records'
 
 export default function ContentWrap({ origin }) {
+  const [records, setRecords] = useState(null)
+  const [dataConnRecordEvents, setDataConnRecordEvents] = useState([])
+  const [statusRecordEvents, errorRecordEvents, startGetRecordsHandler] =
+    useGetLoopedPresentProofRecords()
+
+  useEffect(() => {
+    const setDataFn = (resRecordsData) => {
+      setRecords(resRecordsData)
+      setDataConnRecordEvents((prevData) => {
```

Comment:
setting conn records data very confusing, I think we need simplify

Why do we need to iterate and so much logic, sure it must be a better way

Reply:
reverted ContentWrap change